roles/sap_hana_preconfigure/handlers/main.yml

```diff
@@ -97,3 +97,14 @@
   when:
     - not sap_hana_preconfigure_fail_if_reboot_required | d(true)
     - not sap_hana_preconfigure_reboot_ok | d(false)
+
+- name: GRUB post-update configuration
```

Comment:
@Wabri Can't you use the existing task https://github.com/sap-linuxlab/community.sap_install/blob/b7b10635440eac869a4743fea6eb70053e725dab/roles/sap_hana_preconfigure/handlers/main.yml#L19? It has the same command as this task. Not sure how what we should do with the following task (line 106). For me, it was never required so maybe it's not needed? Otherwise, we should do it for both platforms.

Reply:
Yes, I can, didn't see it until now.